src/figuresClasses.ts

```diff
@@ -1,19 +1,85 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string;
```

Comment:
you can define it here in a such way:

```suggestion
  public shape: string = 'triangle';
```